schema/properties/refined_property.json

```diff
@@ -0,0 +1,25 @@
+{
+    "schemaId": "properties/refined-property",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Schema of material's characteristic.",
+    "allOf": [
+        {
+            "$ref": "raw_property.json"
+        }
+    ],
+    "properties": {
+        "exabyteId": {
+            "description": "Material bank item identity the characteristic obtained for.",
```

Comment:
"Id of the corresponding item in the entity bank that this property is obtained for"